Content.Server/Movement/Systems/JetpackSystem.cs

```diff
@@ -25,10 +27,9 @@ public override void Update(float frameTime)
 
         foreach (var (active, comp, gasTank) in EntityQuery<ActiveJetpackComponent, JetpackComponent, GasTankComponent>())
         {
-            active.Accumulator += frameTime;
-            if (active.Accumulator < UpdateCooldown) continue;
+            if (_timing.CurTime < active.TargetTime) continue;
```

Comment:
For many of these pausing and serialization are being regressed when they shouldn't be. See https://github.com/space-wizards/RobustToolbox/pull/3288 and https://github.com/space-wizards/RobustToolbox/pull/3287